src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts

```diff
@@ -301,14 +269,11 @@ export default class VideoPriorityBasedPolicy implements VideoDownlinkBandwidthP
     // subscribe interval will be changed if probe failed, will take this temporary interval into account
     if (
       noMajorChange &&
-      this.probeFailed &&
       Date.now() - this.lastSubscribeTimestamp < this.timeBeforeAllowSubscribeMs
```

Comment:
Is `this.timeBeforeAllowSubscribeMs` always be `VideoPriorityBasedPolicy.MIN_TIME_BETWEEN_SUBSCRIBE_MS;` now?

Reply:
`timeBeforeAllowSubscribeMs` will be used in probing: if probe failed, this time window will be extended.